### ISSUE TITLE
[Bug] character doubling on locality category

### ISSUE THREAD

**Issue by elRomano** [Cour-de-cassation/label#1045]
## Description
There is a problem on the locality detection. There is a character doubled.
## Screenshot
https://drive.google.com/open?id=1HB9J8sBpeLmtVXehEGDt2hWyuQbkuBUn

**Comment by elRomano** [comment]
@mauryaland I think this one is for you :)

**Comment by mauryaland** [comment]
Yes it is and the good news is that it is fixed and will be push tonight in production!

**Comment by elRomano** [comment]
@mauryaland Great ! Can you check decision #2373248 though. There is the same problem but with an adress...

**Comment by mauryaland** [comment]
It is in production. SIRET number issue is related to location issue. For address it is weird, I will take a look.

**Comment by elRomano** [comment]
Check alerts in admin also there other categories involved

**Comment by mauryaland** [comment]
I do not have any issue on #2373248 with addresses, the index well returned the good text.

**Comment by mauryaland** [comment]
After looking the alerts, it is mainly for localite and etablissement which were concerned by the same bug. It should be good now.

I will add some exception for etablissement to avoid catching stuff like "M." for example.
